webrtc-c/canary/src/CloudwatchMonitoring.cpp

```diff
@@ -13,6 +13,9 @@ STATUS CloudwatchMonitoring::init()
     this->channelDimension.SetName("Channel");
```

Comment:
I know it is not part of this PR, but, to be more specific, can we change this to `WebRTCSDKCanaryChannelName`?

Reply:
Updated to `WebRTCSDKCanaryChannelName`